src/e2e/java/teammates/e2e/util/BackDoor.java

```diff
@@ -235,7 +235,7 @@ public static StudentProfileAttributes getStudentProfile(String userId) {
         params.put(Const.ParamsNames.STUDENT_ID, new String[] { userId });
         JSONObject jsonObj = new JSONObject(executeGetRequest(Const.ResourceURIs.STUDENT_PROFILE, params).responseBody);
 
-        return JsonUtils.fromJson(jsonObj.getJSONObject("studentProfile").toString(), StudentProfileAttributes.class);
```

Comment:
This is changed because?

Reply:
The output of student profile has changed. 
Original was:
```java
class StudentProfileData {
  String name;
  StudentProfileAttribute profile;
}
```

now is 

```java
class StudentProfileData {
  String name;
  String shortName;
  String nationality;
  //...
}
```